cmd/utils/utils.go

```diff
@@ -34,6 +35,7 @@ import (
 // Invoke http-post request using go-resty
 func InvokePOSTRequest(url string, headers map[string]string, body string) (*resty.Response, error) {
 
+    resty.SetTLSClientConfig(&tls.Config{ InsecureSkipVerify: true })
```

Comment:
Can we extract a method for this and call it in line 47 as well to avoid code duplication?